README.md

```diff
@@ -65,6 +65,18 @@ The full spec suite can be run in docker locally. There are several ways to do t
 docker-compose exec web rake
 ```
 
+#### First time spinning up the app
```

Comment:
the shouldn't be needed as docker-compose does this for you. I wonder if you waited until the initialize_app task finished?

Reply:
Valid, can totally pull that.